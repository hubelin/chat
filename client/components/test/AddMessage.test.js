import React from 'react';
import { shallow } from 'enzyme';
import AddMessage from '../AddMessage';

describe('AddMessage', () => {
  let wrapper;
  beforeEach(() => {
    const mockProps = {
      addMessage: jest.fn()
    };
    wrapper = shallow(<AddMessage {...mockProps} />);
  });
  it('renders without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
  it('contains a section for new messages', () => {
    expect(wrapper.find('section#new-message').length).toBe(1);
  });
});
