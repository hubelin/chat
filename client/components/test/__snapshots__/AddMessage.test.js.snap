// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddMessage renders without crashing 1`] = `ShallowWrapper {}`;
